crates/forge/src/scarb.rs

```diff
@@ -43,6 +43,10 @@ pub struct ForgeConfig {
     #[serde(default)]
     /// Should runner exit after first failed test
     pub exit_first: bool,
+    /// How many runs should fuzzer execute
+    pub fuzzer_runs: Option<u32>,
```

Comment:
We should allow configuring this per-case, can we add a task/note for that?

Reply:
#661